doc/reference/reference_lua/decimal.rst

```diff
@@ -18,16 +18,19 @@ to convert the number to decimal type)
 To construct a decimal number, bring in the module with
 ``require('decimal')`` and then use ``decimal.new(n)``
 or any function in the decimal module:
-:ref:`abs(n) <decimal-abs>`
-:ref:`exp(n) <decimal-exp>`
-:ref:`ln(n) <decimal-ln>`
-:ref:`log10(n) <decimal-log10>`
-:ref:`new(n) <decimal-new>`
-:ref:`precision(n) <decimal-precision>`
-:ref:`rescale(decimal-number, new-scale) <decimal-rescale>`
-:ref:`scale(n) <decimal-scale>`
-:ref:`sqrt(n) <decimal-sqrt>`
-:ref:`trim(decimal-number) <decimal-trim>`,
+
+    * :ref:`abs(n) <decimal-abs>`
+    * :ref:`exp(n) <decimal-exp>`
+    * :ref:`is_decimal(n) <decimal-is_decimal>`
+    * :ref:`ln(n) <decimal-ln>`
+    * :ref:`log10(n) <decimal-log10>`
+    * :ref:`new(n) <decimal-new>`
+    * :ref:`precision(n) <decimal-precision>`
+    * :ref:`rescale(decimal-number, new-scale) <decimal-rescale>`
+    * :ref:`scale(n) <decimal-scale>`
+    * :ref:`sqrt(n) <decimal-sqrt>`
+    * :ref:`trim(decimal-number) <decimal-trim>`,
```

Comment:
@Onvember please do not shift the list to the right. The asterisks must be aligned to the paragraph before the list.

```
To construct a decimal number, bring in the module with
``require('decimal')`` and then use ``decimal.new(n)``
or any function in the decimal module:

* :ref:`abs(n) <decimal-abs>`
```